app/src/components/common/layout/header/index.tsx

```diff
@@ -166,6 +166,24 @@ const HeaderDropdown = styled(Dropdown)`
   height: 40px;
 `
 
+const MarketAndGovernanceNav = styled.div<{ disabled?: boolean }>`
+  width: 124.65px;
+
+  background-color: ${props => (props.disabled ? props => props.theme.buttonSecondary.backgroundColor : 'transparent')};
+  ${props => (props.disabled ? 'pointer-events:none;' : '')};
+  border-radius: 32px;
+  color: ${props => (!props.disabled ? props.theme.colors.clickable : props.theme.colors.primary)};
+  cursor: pointer;
+  font-weight: 500;
+  letter-spacing: 1px;
+  line-height: 16.41px;
+  font-size: 14px;
+  text-align: center;
+  padding: 12px 20px;
```

Comment:
font-size and padding vertical and padding horizontal could be imported from the theme.js

Reply:
👍